internal/appconfig/config.go

```diff
@@ -165,6 +174,10 @@ type Compute struct {
 	*api.MachineGuest `toml:",inline" json:",inline"`
 	Processes         []string `json:"processes,omitempty" toml:"processes,omitempty"`
 }
+type Restart struct {
+	Policy     RestartPolicy `toml:"policy,omitempty" json:"policy,omitempty"`
+	MaxRetries int           `toml:"retries,omitempty" json:"retries,omitempty"`
```

Comment:
the thing missing here is the `processes` key as other section have. 

Also as `Restart` is a section, it must go after toplevel keys like `ConsoleCommand`

Reply:
oh, the `processes` thing can be implemented in a following PR if you want to get this merged soonish